launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -193,7 +193,7 @@ void MinecraftInstance::loadSpecificSettings()
     }
 
     // Join server on launch, this does not have a global override
-    m_settings->registerSetting({ "JoinServerOnLaunch", "JoinOnLaunch" }, false);
```

Comment:
doesn't this prevent instances settings saved with old version loading the setting?

Reply:
The old setting is "JoinServerOnLaunch"  the new one is "JoinOnLaunch" but that did not work so going back to the old one as the new one was used only on one line

Reply:
ah, yeah. the that would because the setting "fetch" line uses the alias and not the setting key :P
if it was `m_settings->registerSetting({"JoinOnLaunch",  "JoinServerOnLaunch" }, false);` it should work.

But, honestly, I don't see the point in changing the setting name.